.gitignore

```diff
@@ -3,6 +3,7 @@
 .config
 .rvmrc
 .yardoc
+.ruby-version
```

Comment:
I've added a `.ruby-version` to most projects, with the latest version being supported. It automatically configures multiple development tools and is quite handy that way.